### PR TITLE
Lodash and TweenMax removed - issue #19

### PR DESCRIPTION
**Pull request by lukz**
I did small changes to remove big dependencies - Lodash and TweenMax mentioned in #19 .

### CODE REVIEW

**Comment by mattcolman** [comment]
hey sorry for the slow reply. Definitely like the look of these changes! Planning on reviewing this week.

File: src/basic_swiper.js [hunk 1]
```diff
@@ -75,9 +69,12 @@ export default class BasicSwiper {
       this.clickObject.events.onInputUp.remove(this.handleUp, this)
     }
 
-    forIn(this.events, (signal, key)=> {
-      signal.removeAll()
-    })
+    for (var property in this.events) {
+        if (this.events.hasOwnProperty(property)) {
+            this.events[k].removeAll();
```
Comment (mattcolman) [inline src/basic_swiper.js]: k is not defined.

File: src/scroller.js [hunk 2]
```diff
@@ -107,9 +100,12 @@ export default class Scroller {
       this.clickObject.events.onInputUp.remove(this.handleUp, this)
     }
 
-    forIn(this.events, (signal, key)=> {
-      signal.dispose()
-    })
+    for (var property in this.events) {
+        if (this.events.hasOwnProperty(property)) {
+            this.events[k].dispose();
```
Comment (mattcolman) [inline src/scroller.js]: k is not defined.

**Review by mattcolman — COMMENTED** [review]
@lukz I've updated the examples so when you flick between them the listviews are destroyed. With this you can easily test the destroy methods. I've pointed out in the line comments there is an error in your code. Other than that looks good and will be happy to merge. Thanks!

**Comment by lukz** [comment]
@mattcolman I've fixed those issues. Also there was a small bug in one of examples.

**Pull request merged**